docs/cpp/move-constructors-and-move-assignment-operators-cpp.md

```diff
@@ -125,14 +125,20 @@ The following procedures describe how to write a move constructor and a move ass
     }
     ```
 
-1. In the move assignment operator, add a conditional statement that performs no operation if you try to assign the object to itself.
+1. In the move assignment operator, we need to keep the self-assignment safe, and the simple way is to add a judgment directly.
```

Comment:
I prefer the original wording. 'judgement' and 'safe' are uncommon and ambiguous in technical writing. And 'judgement' wouldn't translate well, either. Can you restore the original?

Reply:
> I prefer the original wording. 'judgement' and 'safe' are uncommon and ambiguous in technical writing. And 'judgement' wouldn't translate well, either. Can you restore the original?

The original description was too absolute.

What I mean here is to ensure "**self-assignment-safe**", but the original text does not mention this key term.

Reply:
I'd like to say "[...], no operation should be performed if one tries to assign the object to itself. The easiest way is adding a conditional statement that skips operations when both operands refer to the same object."